src/main/java/com/limechain/network/Network.java

```diff
@@ -85,7 +85,6 @@ public Network(ChainService chainService, HostConfig hostConfig, KVRepository<St
         this.initializeProtocols(chainService, hostConfig, repository, cliArgs);
         this.bootNodes = chainService.getGenesis().getBootNodes();
         this.chain = hostConfig.getChain();
-        SyncedState.getInstance().setNetwork(network);
```

Comment:
Why is this removed?

Reply:
It's not used anywhere

Reply:
I answered my own question - it's not used. You can resolve this comment